lib/process-services/src/lib/form/widgets/dynamic-table/editors/row-editor/row.editor.ts

```diff
@@ -59,10 +59,10 @@ export class RowEditorComponent {
     column: DynamicTableColumn;
 
     @Output()
-    save: EventEmitter<any> = new EventEmitter<any>();
+    save = new EventEmitter<any>();
```

Comment:
Maybe interface can be added here? Both save and cancel are returning object 
{
    table: this.table,
    row: this.row,
    column: this.column
}
so 
       
export interface RowEditorReturnType {
    table: DynamicTableModel;
    row: DynamicTableRow;
    column: DynamicTableColumn;
}